lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -1341,3 +1353,103 @@ def _capture_libvirt_logs(self) -> None:
             )
             with open(str(libvirt_log_local_path), "w") as f:
                 f.write(libvirt_log)
+
+    def _configure_device_passthrough(
+        self,
+        device_configs: Optional[List[LibvirtHostPciDeviceSchema]],
+        node_context: NodeContext,
+    ) -> None:
+        self._check_passthrough_support(self.host_node)
+        if device_configs:
+            for config in device_configs:
```

Comment:
```suggestion
        if device_configs:
            self._check_passthrough_support(self.host_node)
            for config in device_configs:
```
check for passthrough support has to be done only after `device_configs` check

Reply:
done.